app/src/main/admin/components/ModellingGroups/Models/ModelMetaTable.tsx

```diff
@@ -110,82 +115,88 @@ export class ModelMetaTableComponent extends React.Component<ModelMetaProps, Sta
-                        : "";
-
-                    const outcomesDetailsTooltip = model.outcomes_details.length > 0 ?
-                        createTooltip(`outcomes-details-link-${index}`,
-                            model.outcomes_details.map(function(outcome: Outcome, outcomeIdx: number) {
-                                return (<div key={`outcome-${index}-${outcomeIdx}`}><strong>{`${outcome.code}: `}</strong>{`${outcome.name}`}</div>);
-                            }))
-                        : "";
+                            createTooltip(`countries-details-link-${index}`,
+                                model.countries.map(function (country: Country, countryIdx: number) {
+                                    return (<div key={`country-${index}-${countryIdx}`}>{`${country.name} (${country.id})`}</div>);
+                                }))
+                            : "";
+
+                    const outcomesDetailsTooltip = !obsolete && model.outcomes_details.length > 0 ?
+                            createTooltip(`outcomes-details-link-${index}`,
+                                model.outcomes_details.map(function(outcome: Outcome, outcomeIdx: number) {
+                                    return (<div key={`outcome-${index}-${outcomeIdx}`}><strong>{`${outcome.code}: `}</strong>{`${outcome.name}`}</div>);
+                                }))
+                            : "";
 
                     return (
                         <tr key={index} data-item={model}>
                             <td data-title="group">{model.modelling_group}</td>
                             <td data-title="name">{model.id}</td>
                             <td data-title="disease">{model.disease}</td>
                             <td data-title="type">{model.is_dynamic ? "Dynamic" : "Static"}</td>
-                            <td data-title="scenarios">{`${model.scenario_count} scenario` +
+                            {obsolete ||
+                                <td data-title="scenarios">{`${model.scenario_count} scenario` +
                                                             (model.scenario_count === 1 ? "" : "s")}
                                 {scenarioDetailsLink}
-                            </td>
+                                </td>}
                             <td data-title="code">{model.code}</td>
                             <td data-title="gender">{model.gender ? model.gender : "NA"}</td>
                             <td data-title="max_countries">{model.max_countries}
                                 {countriesDetailsLink}</td>
-                            <td data-title="years">{model.years}</td>
-                            <td data-title="ages">{model.ages}</td>
-                            <td data-title="cohorts">{model.cohorts}</td>
-                            <td data-title="outcomes">{model.outcomes}
+                            {obsolete || <td data-title="years">{model.years}</td>}
```

Comment:
Maybe the creation of these 5 columns could be moved into their own method so this is a little easier to parse?
So you end up with something like `{obsolete} || expecationsColumns()` instead of repeating the conditional. Similarly for the corresponding headers.